content/faq.md

```diff
@@ -114,10 +114,10 @@ title: Frequently Asked Questions
   the bandwidth associated with downloading the large file many times can be
   saved.  The reference implementation provides an [easy way to distribute
   target files across many targets
-  metadata](https://github.com/theupdateframework/python-tuf/blob/d4b308ae13acfe832bfb7f993108e5e065d44c04/tuf/repository_tool.py#L2387)
-  (i.e., delegating to hashed bins), which the
-  [specification](https://github.com/theupdateframework/specification/blob/master/tuf-spec.md)
-  refers to as path hash prefixes.
+  metadata](https://github.com/theupdateframework/python-tuf/blob/v0.20.0/examples/repo_example/hashed_bin_delegation.py)
+  (i.e., delegating to hashed bins), which the specification refers to as [path
```

Comment:
Outside the scope of this PR, but we should resolve that we generally refer to this as hashed bin delegation, but the spec doesn't use that term. See somewhat related issue https://github.com/theupdateframework/specification/issues/109

Reply:
Agreed. The about to be deprecated [python tuf tutorial](https://github.com/theupdateframework/python-tuf/blame/v0.11.1/docs/TUTORIAL.md#L584) and until recently  [PEP458](https://github.com/python/peps/pull/2240/files#diff-8b6e17539bd49f8b8c9563f2484ce03feceaeaee02a916637f068493c9ae5410L614) in addition use the term "lazy bin walk".